src/pages/discovery/loadingAnimation/loadingAnimation.scss

```diff
@@ -0,0 +1,15 @@
+#discovery-loader{
```

Comment:
This all generally looks good. Just needs to be updated with new general ID values once it is moved to components folder. same with below class names